cmd/util/cmd/execution-state-extract/execution_state_extract.go

```diff
@@ -120,11 +120,16 @@ func extractExecutionState(
 	if exportPayloads {
 		payloads := newTrie.AllPayloads()
 
+		// Sort payloads to produce deterministic payload file with
+		// same sequence of payloads inside.
+		payloads = util.SortPayloadsByAddress(payloads, nWorker)
+
 		exportedPayloadCount, err := util.CreatePayloadFile(
 			log,
 			outputPayloadFile,
 			payloads,
 			exportPayloadsByAddresses,
+			false, // payloads represents entire state.
```

Comment:
Maybe not in this case because most of the non-test calls of `CreatePayloadFile()` uses variable returned from another function call (not hard-coded) to indicate if input payloads are for partial or full state.  So If we split it into two functions, we need to check this boolean and decide which function to call at higher level.